src/contrib/persistence/Akka.Persistence.Sql.Linq2Db/Journal/Types/JournalRow.cs

```diff
@@ -0,0 +1,27 @@
+﻿using LinqToDB;
+using LinqToDB.Mapping;
+
+namespace Akka.Persistence.Sql.Linq2Db.Journal.Types
+{
+    public sealed class JournalRow
+    {
+        [Column(Configuration = ProviderName.SQLite, DbType = "INTEGER", IsIdentity = true, IsPrimaryKey = true)]
+        [Column(IsIdentity = true, IsPrimaryKey = false)]
+        public long ordering { get; set; }
+        
+        public long Timestamp { get; set; } = 0;
+
+        public bool deleted { get; set; }
+        [Column(Configuration = ProviderName.SQLite, IsPrimaryKey = false)]
+        [Column(IsPrimaryKey = true, CanBeNull = false)]
+        public string persistenceId { get; set; }
+        [Column(Configuration = ProviderName.SQLite, IsPrimaryKey = false)]
+        [Column(IsPrimaryKey = true)]
+        public long sequenceNumber { get; set; }
```

Comment:
in SQLite, if you have an auto-increment sequence, it MUST be the primary key, so this is a tiny bit hacky since linq2db doesn't let us specify a unique constraint

Reply:
This has been changed now to be in the mapping